### PR TITLE
Add support for National Identifiers

### PR DESCRIPTION
**Pull request by plamenxpenchev**

### CODE REVIEW

**Review by rveda — APPROVED** [review]
Love that you have documented the constraints as well. :+1:

File: version1/02 - Object types.md [hunk 1]
```diff
@@ -480,6 +485,7 @@ The following notation is used in the tables below:
...
 |           | ref       | refMerknad                       |          |             |                 |  |      |      |   x   |      |  x   |   x    |                                                                                                                                                                                                                                                                                                        |
+|           | ref       | refNasjonalIdentifikator         |          |             |                 |  |      |      |   x   |      |  x   |   x    |                                                                                                                                                                                                                                                                                                        |
```
Comment (dkumanov) [inline version1/02 - Object types.md]: We cannot unlink the national identifier from the registry entry. We also have mistakenly marked that we can unlink notes and sign-offs above.
```suggestion
|           | ref       | refNasjonalIdentifikator         |          |             |                 |  |      |      |   x   |      |  x   |        |                                                                                                                                                                                                                                                                                                        |
```

File: version1/02 - Object types.md [hunk 2]
```diff
@@ -609,6 +615,102 @@ The following notation is used in the tables below:
 |           | ref       | refDokument            |    x     |             |               |  |      |  x   |   x   |      |  x   |   x    |                                                                    |
 |           | ref       | refEksternId           |          |             |               |  |      |      |   x   |      |  x   |        | &nbsp;                                                             |
 
+## National Identifiers
```
Comment (dkumanov) [inline version1/02 - Object types.md:626]: Each type can also have the `beskrivelse` (description) property.

**Review by dkumanov — CHANGES_REQUESTED** [review]

**Comment by teodoradobreva** [comment]
We should probably also describe national identifiers in the `### Abstract object types` section in a similar way to folders/records.

File: version1/02 - Object types.md [hunk 3]
```diff
@@ -609,6 +615,102 @@ The following notation is used in the tables below:
...
+|           | string    | nummer                 |          |             |               |  |  x   |  x   |   x   |  x   |      |        |                                                                    |
+|           | string    | adressekode            |          |             |               |  |  x   |  x   |   x   |  x   |      |        |                                                                    |
+|           | string    | alternativAdresse      |          |             |               |  |  x   |  x   |   x   |  x   |      |        |                                                                    |
+|           |           |                        |          |             |               |  |      |      |       |      |      |        |                                                                    |
+|           | ref       | refMappe               |    c     |             |               |  |      |  x   |   x   |      |  x   |        | <p>&bull; references **AbstraktMappe**</p><p>&bull; exactly one of **refMappe** and **refRegistrering** is required</p>        |
```
Comment (teodoradobreva) [inline version1/02 - Object types.md]: Table formatting is a bit off in all national identifier tables.

**Comment by plamenxpenchev** [comment]
> We should probably also describe national identifiers in the `### Abstract object types` section in a similar way to folders/records.

Addressed it with [this commit](https://github.com/documaster/noark5-web-services/pull/37/commits/51333815133c08ab5a275135feca0bd489411c78).

File: version1/02 - Object types.md (hunk 2 above)
Comment (plamenxpenchev) [reply version1/02 - Object types.md:626]: Addressed with [this commit](https://github.com/documaster/noark5-web-services/pull/37/commits/01b71b03b04b6a811a1355472d58ff30d84e2da7).

File: version1/02 - Object types.md (hunk 3 above)
Comment (plamenxpenchev) [reply version1/02 - Object types.md]: Addressed with [this commit](https://github.com/documaster/noark5-web-services/pull/37/commits/1d250c1a3df5b0886e8541db740d116c0064771b).

**Comment by plamenxpenchev** [comment]
@dkumanov applied the change for the falsely specified unlink possibility for Journalpost with [this commit](https://github.com/documaster/noark5-web-services/pull/37/commits/881c81ae3533ade25b2711b9feae7b7b115e7687).

Tested unlinking with all types:
`
{
  "actions": [
    {
      "action": "unlink",
      "type": "NasjonalIdentifikator",
      "id": 795384,
      "ref": "refRegistrering",
      "unlinkFromId": ["795378"]
    }
  ]
}
`
and the result was
`no.dots.exceptions.link.InvalidLinkUnlinkOperationException: Unlinking the object from its parent is not allowed; Affected objects types: [NationalIdentifier type] and [Record type]`

**Comment by plamenxpenchev** [comment]
All current comments have been addressed.

**Review by dkumanov — APPROVED** [review]

**Pull request merged**